### ISSUE TITLE
When I click on the file that grabs your token it deletes itself

### ISSUE THREAD

**Issue by XXfusion7** [Rdimo/Hazard-Token-Grabber-V2#490]

**Comment by GodFather227** [comment]
turn off ur anti virus

**Comment by Rdimo** [comment]
yes, turn off the "hide_self" option in main.py if you don't want it to do that

**Comment by heckair** [comment]
where is going the file after ? Does he will automaticly start after reboot ?

**Comment by dekrypted** [comment]
> where is going the file after ? Does he will automaticly start after reboot ?

Yes! The file isn't deleted, it's just hidden. If you enable "hidden items" in the view menu you will be able to see it.
